rfc/networked-dot-storage.md

```diff
@@ -0,0 +1,174 @@
+# Networked Dot Storage
+
+> or what does decentralization even mean?
+
+## Authors
+
+- [Hannah Howard], [Protocol Labs]
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## Abstract
+
+This RFC puts forward a proposal for the broad parameters of "decentralization" in W3S. It defines both the goals for "decentralization" and identifies where resources should be invested first on the path to "decentralization".
+
+## Non-goals
+
+1. Addressing anything related to tokens
+2. W3S corporate structure or nucleation plans
+
+## Background
+
+The current state of the world for web3.storage is as follows:
+
+- Web3.storage UCAN protocols are open and well specified. 
+
+- We've now created a version of these protocols that can be run from any machine-- so anyone can spin up theoretically own version of our service.
+
+- Seperately Web3.storage backs up data to Filecoin, via open protocols.
+
+- We are actively moving towards supporting running on top of underlying storage mechanisms that lack rich event functionality (a.k.a. bucket events)
+
+## Why Go Further
+
+Today, someone who is not the web3.storage team could spin up their own version of web3.storage, using the open source software we've published and the protocols we've specified. No one has done this in practice, but we are increasingly close to the point where it's plausable. Is this "sufficient decentralization"?
+
+One proposed path forward is to not pursue "decentralization" beyond trying to make other nodes a reality, perhaps through incentivization, and leaning heavily into product improvements that differentiate our platform. We could aim for "S3 if AWS invested in open source S3 implementations and made it content addressed".
+
+This is not the path we are going to pursue. It feels important to say that up front.
```

Comment:
> This is not the path we are going to pursue.

Well, it's funny - what's described below is a lot closer to what I meant by "S3 if AWS invested in open source S3 implementations and made it content addressed" than what I understood we were aiming to build, and the differences are things I feel pretty positive about - coordinating different w3up operators in order to make the user experience really good, exposing more of the details of what Filecoin is providing (eg, teaching the end user about the proofs that their data is replicated across the Filecoin network and exposing them in a user friendly way in our UI), data portability and low barriers to entry to getting a node running, etc

tbh after reading this line I was expecting to be disappointed by the rest of this doc but it's just the opposite - maybe just a masterful use of expectations ;-p 

anyway TLDR I'm pretty positive about this direction, thanks for putting this together! will leave more thoughts inline below...

Reply:
That's fair. 

I will say something, which I think many people are talking about similar underlying things through different language, leading to a perception of difference in goals that is outsized (even if there are real, if smaller differences underneath). My thought is that describing user goals is what can produce better conversations.

Reply:
> people are talking about similar underlying things through different language, leading to a perception of difference in goals that is outsized 

agree 100% - super grateful for this document as I think it does a great job of 

> describing user goals is what can produce better conversations.